mysql/tweets_table.sql

```diff
@@ -1,14 +1,14 @@
--- tweetsテーブルを作成
-CREATE TABLE tweets (
+-- Tweetsテーブルを作成
+CREATE TABLE Tweets (
   id INT AUTO_INCREMENT NOT NULL PRIMARY KEY,
   user_id INT NOT NULL,
   title VARCHAR(30),
   tweet_content VARCHAR(140),
-  FOREIGN KEY fk_user_id(user_id) REFERENCES users(id) -- user_idカラムとusersテーブルのidカラムを紐付け
+  FOREIGN KEY fk_user_id(user_id) REFERENCES Users(id) -- user_idカラムとUsersテーブルのidカラムを紐付け
 );
 
--- tweetsに挿入
-INSERT INTO tweets
+-- Tweetsに挿入
+INSERT INTO Tweets
```

Comment:
Idiomatic

テーブル名は小文字のアルファベットで始まるのが慣習的かな。
```suggestion
INSERT INTO tweets
```